src/lambda/commands/createNewSamApp.ts

```diff
@@ -200,8 +201,15 @@ export async function createNewSamApplication(
             true
         )
 
-        await addInitialLaunchConfiguration(extContext, vscode.workspace.getWorkspaceFolder(uri)!, uri)
+        const newLaunchConfigs = await addInitialLaunchConfiguration(
```

Comment:
Unless VS Code has changed recently, when `addFolderToWorkspace` runs, there are circumstances where VS Code closes and re-launches itself. One example was when creating a SAM app when starting with no open workspace.

Is the code that was added below guaranteed to run, or does it need to be relocated to `resumeCreateNewSamApp` ?

Reply:
It wouldn't be guaranteed to run; question is, do we want it to run on changing the workspace folder?

Reply:
I'll check for sure but at this time it looks like the launch.json file won't be created on `addFolderToWorkspace`. I'm thinking that this is out of scope for this PR but we should move both of those functionalities (the launch config creation and the notification) to the `resumeCreateNewSamApp` functionality.

Reply:
Agreed, I initially thought this PR introduced the launch config as well.